src/elm/Page/Community/Settings/Currency.elm

```diff
@@ -0,0 +1,560 @@
+                    div [ class "bg-white" ]
+                        [ Page.viewHeader loggedIn title Route.CommunitySettings
+                        , view_ loggedIn community model
+                        ]
+    in
+    { title = title
+    , content = content
+    }
+
+
+view_ : LoggedIn.Model -> Community.Model -> Model -> Html Msg
+view_ { shared } community model =
+    let
+        { t } =
+            shared.translators
+
+        precision =
+            Eos.getSymbolPrecision community.symbol
+
+        fillWithPrecision amount =
+            if precision == 0 then
+                String.fromInt amount
+
+            else
+                String.fromInt amount ++ "." ++ String.join "" (List.repeat precision "0")
+    in
+    form
+        [ class "w-full px-4 pb-10"
+        , onSubmit ClickedSubmit
+        ]
+        [ div [ class "container mx-auto pt-4" ]
+            [ Input.init
+                { label = t "community.create.labels.currency_name"
+                , id = "currency_name_field"
+                , onInput = \_ -> Ignored
+                , disabled = True
+                , value = community.name
+                , placeholder = Nothing
+                , problems = Nothing
+                , translators = shared.translators
+                }
+                |> Input.toHtml
+            , div [ class "flex w-full space-x-8" ]
+                [ div [ class "w-full" ]
+                    [ Input.init
+                        { label = t "community.create.labels.currency_symbol"
+                        , id = "currency_symbol_field"
+                        , onInput = \_ -> Ignored
+                        , disabled = True
+                        , value = Eos.symbolToSymbolCodeString community.symbol
+                        , placeholder = Nothing
+                        , problems = Nothing
+                        , translators = shared.translators
+                        }
+                        |> Input.toHtml
+                    ]
+                , div [ class "w-full" ]
+                    [ Input.init
+                        { label = t "settings.community_currency.decimal_places"
+                        , id = "currency_precision_field"
+                        , onInput = \_ -> Ignored
+                        , disabled = True
+                        , value = String.fromInt precision
+                        , placeholder = Nothing
+                        , problems = Nothing
+                        , translators = shared.translators
+                        }
+                        |> Input.withAttrs [ class "w-full" ]
+                        |> Input.toHtml
+                    ]
+                ]
+            , div [ class "bg-gray-100 py-4 text-center mb-10" ]
+                [ div [ class "text-xl font-medium space-x-4 mb-4" ]
+                    [ span [ class "text-black" ]
+                        [ text
+                            (String.fromFloat pi
```

Comment:
I like `pi` but I've found it to be confusing. @heltonlr do you think we can improve this?
<img width="523" alt="Captura de Tela 2021-04-19 às 15 09 57" src="https://user-images.githubusercontent.com/1082127/115283083-4e9f7d80-a121-11eb-92d3-142078e7a29f.png">


I mean I know that we can have more decimal places, but it might look confusing to see this: 3 MIZU your currency format

Reply:
@heltonlr ping

Reply:
Ping @rafachadud

Reply:
Oi @lucca65 , the number 3 is an example of what a value looks like in the configuration that the user made, in the case of 3 MIZO (which has 0 decimal places). What were you confused about?

Reply:
Yeah I know! I mean we are already super familiar with the whole flow and we know about precision! What I'm trying to say is that this screen configured with 0 don't gives the idea that this is a token without decimal cases. Maybe we can show pi but only highlight the available decimal cases (in this case 0)